### ISSUE TITLE
How to obtain validation loss without training again?

### ISSUE THREAD

**Issue by ortegalobo** [qqwweee/keras-yolo3#679]

**Comment by datduonguva** [comment]
There is a tensorboard log. Just do
 tensorboard --logdir <dir where you put your tensorboard log>
Then you can find the history of the whole training process
